comictaggerlib/renamewindow.py

```diff
@@ -82,7 +84,11 @@ def config_renamer(self, ca: ComicArchive, md: GenericMetadata | None = None) ->
             new_ext = ca.extension()
 
         if md is None or md.is_empty:
-            md = ca.read_metadata(self.data_style)
+            md, error = self.parent().overlay_ca_read_style(self.load_data_styles, ca)
+            # How much should we care about this error? Notify user or not?
+            if error is not None:
+                logger.error("Failed to load metadata for %s: %s", ca.path, error)
+
```

Comment:
Should the user be informed in a window or is it not all that relevant here?

Reply:
The user should have already been informed but we should still let them know here